compiler/damlc/tests/daml-test-files/ActionFail.daml

```diff
@@ -1,12 +1,12 @@
 -- Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
 -- All rights reserved.
 
--- @INFO range=9:21-9:38; Use catOptionals
-
 module ActionFail where
 
+import DA.Optional
+
 catOptionalsComp : [Optional a] -> [a]
-catOptionalsComp l = [x | Some x <- l]
+catOptionalsComp = catOptionals
```

Comment:
This doesn’t make sense. We want to test the list comprehension here.

Reply:
Ok